app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```

Comment:
Code Style: It's a good practice to use type annotations for all inputs and outputs of your functions. Here you should annotate 'friends' and 'cafe'.